### ISSUE TITLE
Spacing between brace and closure parameters

### ISSUE THREAD

**Issue by maveric94** [realm/SwiftLint#2186]
### New Issue Checklist

- [ ✔️] Updated SwiftLint to the latest version
- [ ✔️] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Rule Request

Rule to check spacing between opening brace of closure and following parameters. Maybe it can be added to closure_parameter_position rule.

### Environment

* SwiftLint version 0.25.1
* CocoaPods

* Xcode version 9.3

```swift
// This doesn't triggers a violation:
let test: (Bool) -> Void = {          obj in
}
```

**Comment by Reggian** [comment]
The initial space between opening brace and first closure parameter could also be added to `closure_spacing` rule (which currently handles only single lined closures).
